lib/api/controllers/baseController.js

```diff
@@ -76,18 +78,18 @@ class BaseController {
    * @param {string} flagPath
    */
   tryGetBoolean (request, flagPath) {
-    const
-      flagName = flagPath.split('.').slice(-1),
-      flagValue = _.get(request, `input.${flagPath}`);
+    const flagValue = _.get(request, `input.${flagPath}`);
```

Comment:
This review seems to have been on an older version of that branch: the method `tryGetBoolean` doesn't exist since the [optimization commit](https://github.com/kuzzleio/kuzzle/pull/1481/commits/984ccd19c8981571101c538a721198b128a8abba)